### ISSUE TITLE
create directories from config file

### ISSUE THREAD

**Issue by annakozyreva1** [go-ozzo/ozzo-log#13]
I want to ask to add it creating folders from config file if it is not exist. 
for example: 
"type": "FileTarget",
"FileName": "c:/sed/1.log"
the library creates c:\sed

To rewrite localy the library is bad decision.

**Comment by kolkov** [comment]
Makes sense — auto-creating parent directories for the log file path is a reasonable default. Most logging libraries do `os.MkdirAll` before opening the file. It's a small change and I agree that forking just for this isn't ideal.

I'm the new maintainer here and will include this in an upcoming patch. Thanks for the suggestion.
